src/models/adrPassCertificate.ts

```diff
@@ -0,0 +1,85 @@
+import { ADRCertificateTypes } from '@dvsa/cvs-type-definitions/types/v3/tech-record/enums/adrCertificateTypes.enum.js';
+import { DocumentName } from '../enums/documentName.enum';
+import { DocumentModel } from './documentModel';
+import { Request } from './request';
+
+export type AdrCert = {
+  vin: string;
+  make: string;
+  vrm: string;
+  applicantDetails: {
+    name?: string;
+    address1?: string;
+    address2?: string;
+    postTown?: string;
+    address3?: string;
+    postCode?: string;
+    telephoneNumber?: string;
+  };
+  adrVehicleType: string;
+  permittedDangerousGoods: string[];
+  brakeEndurance: boolean;
+  weight: string;
+  tankManufacturer: string;
+  tc2InitApprovalNo: string;
+  tankManufactureSerialNo: string;
+  yearOfManufacture: string;
+  tankCode: string;
+  specialProvisions: string;
+  tankStatement: {
+    substancesPermitted: string;
+    statements: string;
+    productList: string;
+  };
+  notes: string;
+  replacement: boolean;
+};
+
+export class AdrPassCertificateDocument extends DocumentModel {
+  constructor(request: Request) {
+    super('');
+    const { techRecord, adrCertificate } = request;
+
+    this.setDocumentType(DocumentName.ADR_PASS_CERTIFICATE);
+    this.filename = `adr_pass_${techRecord.systemNumber}_${adrCertificate.generatedTimestamp}`;
+
+    // S3 metadata
+    this.metaData.vin = techRecord.vin;
+    this.metaData['should-email-certificate'] = 'false';
+
+    const adrData: AdrCert = {
+      // ADR data
+      vin: techRecord.vin,
+      make: techRecord.techRecord_make,
+      vrm: techRecord.techRecord_vehicleType === 'hgv' ? techRecord.primaryVrm : techRecord.trailerId,
```

Comment:
```suggestion
      vrm: techRecord.techRecord_vehicleType === 'trl' ? techRecord.trailerId : techRecord.primaryVrm,
```
I think we need the ternary this way round to support lgv?

Reply:
yeah just did this, lol